apis/apps/v1beta3/emqxbroker_types.go

```diff
@@ -38,38 +43,84 @@ type EmqxBrokerTemplate struct {
 	// See https://github.com/emqx/emqx-operator/pull/72
 	ExtraVolumeMounts []corev1.VolumeMount `json:"extraVolumeMounts,omitempty"`
 
+	// Config represents the configurations of EMQX cluster
+	// More info: https://www.emqx.io/docs/en/v4.4/configuration/configuration.html#cluster
 	EmqxConfig EmqxConfig `json:"config,omitempty"`
-	Args       []string   `json:"args,omitempty"`
-
-	SecurityContext *corev1.PodSecurityContext  `json:"securityContext,omitempty"`
-	Resources       corev1.ResourceRequirements `json:"resources,omitempty"`
-
+	// Args define arguments for the command
+	Args []string `json:"args,omitempty"`
```

Comment:
Arguments to the entrypoint. The container image's CMD is used if this is not provided. Variable references $(VAR_NAME) are expanded using the container's environment. If a variable cannot be resolved, the reference in the input string will be unchanged. Double $$ are reduced to a single $, which allows for escaping the $(VAR_NAME) syntax: i.e. "$$(VAR_NAME)" will produce the string literal "$(VAR_NAME)". Escaped references will never be expanded, regardless of whether the variable exists or not. Cannot be updated. More info: https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#running-a-command-in-a-shell